caraml-store-sdk/python/feast/maxcompute/staging.py

```diff
@@ -0,0 +1,102 @@
+import os
+import uuid
+import pandas as pd
+from datetime import datetime, timedelta
+from feast.data_source import MaxComputeSource
+from odps import ODPS
+from odps import df
```

Comment:
should we put odps in requirements-ci.txt?

Reply:
fixed